etc/eslint/rules/typescript.js

```diff
@@ -0,0 +1,2526 @@
+*
+* @name explicit-function-return-type
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [explicit-function-return-type]{@link https://typescript-eslint.io/rules/explicit-function-return-type}
+*
+* @example
+* // Bad...
+* function test() {
+*   return 42;
+* }
+*
+* // Good...
+* function test(): number {
+*   return 42;
+* }
+*/
+rules[ '@typescript-eslint/explicit-function-return-type' ] = 'error';
+
+/**
+* Enforces explicit visibility declarations for class members.
+* 'no-public' option requires explicit visibility except for public members.
+*
+* @name explicit-member-accessibility
+* @memberof rules
+* @type {Array}
+* @default [ 'error', {'accessibility': 'no-public'}]
+* @see [explicit-member-accessibility]{@link https://typescript-eslint.io/rules/explicit-member-accessibility}
+*
+* @example
+* // Bad...
+* class MyClass {
+*   memberWithoutVisibility = 'no explicit visibility';
+* }
+*
+* // Good...
+* class MyClass {
+*   private memberWithVisibility = 'has explicit visibility';
+* }
+*/
+rules[ '@typescript-eslint/explicit-member-accessibility' ] = [
+	'error',
+	{
+		'accessibility': 'no-public'
+	}
+];
+
+/**
+* Requires explicit return and argument types on exported functions' and classes' public class methods.
+*
+* @name explicit-module-boundary-types
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [explicit-module-boundary-types]{@link https://typescript-eslint.io/rules/explicit-module-boundary-types}
+*
+* @example
+* // Bad...
+* export function badFunction(a) {
+*   return a;
+* }
+*
+* // Good...
+* export function goodFunction(a: string): string {
+*   return a;
+* }
+*/
+rules[ '@typescript-eslint/explicit-module-boundary-types' ] = 'error';
+
+/**
+* Enforces indentation with tabs.
+*
+* @name indent
+* @memberof rules
+* @type {Array}
+* @default [ 'error', 'tab' ]
+* @see [indent]{@link https://typescript-eslint.io/rules/indent}
+*
+* @example
+* // Bad...
+* function bad() {
+*  return true;
+* }
+*
+* // Good...
+* function good() {
+*   return true;
+* }
+*/
+rules[ '@typescript-eslint/indent' ] = [ 'error', 'tab' ];
+
+/**
+* Enforces a specific member delimiter style in interfaces and type literals.
+*
+* @name member-delimiter-style
+* @memberof rules
+* @type {Array}
+* @default [ 'error', { 'multiline': { 'delimiter': 'semi', 'requireLast': true }, 'singleline': { 'delimiter': 'semi', 'requireLast': false } }]
+* @see [member-delimiter-style]{@link https://typescript-eslint.io/rules/member-delimiter-style}
+*
+* @example
+* // Bad...
+* interface Bad {
+*   name: string,
+*   age: number
+* }
+*
+* // Good...
+* interface Good {
+*   name: string;
+*   age: number;
+* }
+*/
+rules[ '@typescript-eslint/member-delimiter-style' ] = [
+	'error',
+	{
+		'multiline': {
+			'delimiter': 'semi',
+			'requireLast': true
+		},
+		'singleline': {
+			'delimiter': 'semi',
+			'requireLast': false
+		}
+	}
+];
+
+/**
+* Enforces member ordering.
```

Comment:
This likely won't affect us, as we don't do `class` development in TypeScript. In JavaScript, we don't follow this rule.

Reply:
Let's just remove it then.